### PR TITLE
hi to whoever's reading this. cheers m8

### PR DESCRIPTION
**Pull request by ghost**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

[here :D although i just want it for emails](https://saxophone-guy.github.io/website/)

### CODE REVIEW

**Comment by saxophone-guy** [comment]
whoops!
wrong domain lol. closing and opening a new PR

**Review by iostpa — CHANGES_REQUESTED** [review]
Please add more stuff to your website

**Comment by saxophone-guy** [comment]
Sure. I'll upload the finished ver. today

**Comment by saxophone-guy** [comment]
Alright. Added a webgl demo to the page. Here's a link: https://saxophone-guy.github.io

**Comment by CuteDog5695** [comment]
> Alright. Added a webgl demo to the page. Here's a link: https://saxophone-guy.github.io
> 
> Not sure if it needs to be like a full on website tho. I just need to use it for emails.

Then you can just add MX and TXT records?

File: domains/saxophone.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "username": "saxophone-guy",
+    "email": "itsarchit@keemail.me"
+  },
+  "record": {
+    "CNAME": "saxophone-guy.github.io",
+    "MX": [
+      "mx1.forwardemail.net",
+      "mx2.forwardemail.net"
+    ],
+    "TXT": "forward-email=itsarchit@keemail.me"
```
Comment (CuteDog5695) [inline domains/saxophone.json:13]: You cannot use CNAME with any other records

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/saxophone.json (hunk 1 above)
Comment (saxophone-guy) [reply domains/saxophone.json:13]: Done!

Comment (CuteDog5695) [reply domains/saxophone.json:13]: You didn't do anything though?

Comment (saxophone-guy) [reply domains/saxophone.json]: Sorry about that! I edited my changes locally but forgot to actually push. Please check again.

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**